[SNAP-1624] initialize ResultSetTiming for global index scan

sumwale opened the pull request:
Two issues, a product issue of missing timing for global index and a test issue where the test
should not have global index scan.

## Changes proposed in this pull request

- timing statistic for global index scan was created but not assigned
- index itself was an artifact from a previous test so cleaning it up
- also enable conflation missing in some stop/remove system procedures

## Patch testing

store precheckin

## ReleaseNotes changes

NA

## Other PRs 

NA

Commit:
[SNAP-1624] initialize ResultSetTiming for global index scan

Two issues, a product issue and a test issue:

- timing statistic for global index scan was created but not assigned
- index itself was an artifacts from a previous test so cleaning it up

vibhaska commented on gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/stats/StatementPlanDUnit.java:
```diff
@@ -748,7 +749,10 @@ public void testBug43228() throws Exception {
+          conn, "test_globalindex_plan_check")) {
+        st.execute("drop table test_globalindex_plan_check");
+      }
     }
```
Good to close st and conn

Commit:
review comment

also fix occasional failure in IndexPersistenceDUnit due to moving buckets after restart

Commit:
Fixing some issues in DDL queue conflation

- don't conflate two DROP statements
- correct the return value of GfxdOpConflationHandler.applyConflation to
  return true if any of the passed lists were populated
- correct getRegionToConflate() for sys proc messages (GfxdGatewaySenderStart/StopMessage,
    GfxdSet/RemoveGatewayEventErrorHandlerMessage, GfxdSet/RemoveGatewayConflictResolverMessage)
- remove config-scripts completely from code and its remaining unit test which is
  a broken design (inconsistent DDL queues) and unsupported for quite long
